config/vufind/EDSRecordDataFormatter.ini

```
@@ -0,0 +1,69 @@
+; This file is equivalent to RecordDataFormatter.ini, but is applied to EDS records instead of Solr records.
+; If using this is not sufficient you can also adapt the specs by overriding RecordDataFormatter/Specs/EDS.
```

Would this be more accurate?

```suggestion
; If using this is not sufficient you can also adapt the specs by overriding RecordDataFormatter\Specs\EDS
; in the RecordDataFormatter\Specs\PluginManager service.
```